README.md

```diff
@@ -128,7 +128,7 @@ debug = true
 As usual, thanks to Clap, running `cargo instruments -h` prints the compact help.
 
 ```
-cargo-instruments 0.4.5
+cargo-instruments 0.4.7
```

Comment:
I copied-and-pasted the literal output from `cargo instruments -h` so looks like a couple other things were slightly out-of-date as well